models/resetToken.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const tokenSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+  token: {
+    type: String,
+    required: true,
+  },
```

Comment:
what is the actual requirement of two model for same things `resetToken.js` and `access_token.js` ?